src/AnalyticsAdapter.js

```diff
@@ -130,6 +137,7 @@ export default function AnalyticsAdapter({ url, analyticsType, global, handler }
     utils._each(_handlers, (handler, event) => {
       events.off(event, handler);
     });
+    this.enableAnalytics = this._oldEnable ? this._oldEnable : _enable;
```

Comment:
i'm not clear on what this change is for.

Reply:
`enableAnalytics` is overwritten w/ the error message function above when an analytics adapter is enabled.  This new code I added sets `enableAnalytics` back to the correct enable function when analytics is disabled (can't just set it back to `_enable` as some analytics extend `enalbeAnalytics` with a new function).  

Probably not a use case you'd encounter much in production (disabling analytics and then enabling again), but in test cases it was an issue. Many of the analytics adapters were causing issues with other test suites as they were not cleaning up their event handlers, probably in part because cleaning them up (using `disableAnalytics`) didn't allow you to listen again since `enableAnalytics` was overwritten; this fixes that.